src/views/InProgress.vue

```diff
@@ -711,6 +711,13 @@ export default defineComponent({
       return defaultBoxType;
     },
     async addShipmentBox(order: any) {
+      if(order.isAddingBox) {
```

Comment:
We should add toast message that already an action is going on

Reply:
Discussed we will disable the button instead of adding a toast.